.github/workflows/version_bump.yml

```
@@ -0,0 +1,49 @@
+name: Version bump
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 * * * *"
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
+    env:
+      REPO: ${{ github.repository }}
+      GH_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+    steps:
+      - name: Get latest release tag # can also be a draft release
+        run: |
+          LATEST_TAG=$(gh -R $REPO release list -L 1 |awk '{printf $3}')
```

```suggestion
          LATEST_TAG=$(gh -R $REPO release list -L 1 | awk '{printf $3}')
```

Splitting hairs, I know...

---

😜